scripts/configuration/ac-build-config.xml

```diff
@@ -2,7 +2,7 @@
 <config>
     <sdkInfo>
         <sdkVersion>1.13.2-SNAPSHOT</sdkVersion>
-        <iosVersion>1.13.0</iosVersion>
+        <iosVersion>1.13.2</iosVersion>
         <androidVersion>1.11.2</androidVersion>
```

Comment:
https://github.com/Microsoft/AppCenter-SDK-Android/releases => we didn't bump to 1.11.3 last time, was that missed on first release attempt?

Reply:
Yes

Reply:
Good thing we see it now then.